pygsuite/common/style.py

```diff
@@ -76,6 +93,69 @@ def to_doc_style(self) -> Tuple[str, Dict]:  # noqa: C901
 
         return ",".join(masks), base
 
+    def to_sheet_style(self) -> Tuple[list, Dict]:  # noqa: C901
+
+        base = {}
+        masks = []
+
+        if self.font_size is not None:
+            base["fontSize"] = {"magnitude": self.font_size, "unit": "PT"}
+            masks.append("fontSize")
+        if self.bold is not None:
+            base["bold"] = self.bold
+            masks.append("bold")
+        if self.italic is not None:
+            base["italic"] = self.italic
+            masks.append("italic")
+        if self.underline is not None:
+            base["underline"] = self.underline
+            masks.append("underline")
+        if self.color is not None:
+            base["foregroundColor"] = self.color.to_json()
+            masks.append("foregroundColor")
+        if self.font is not None:
+            base["fontFamily"] = self.font
+            masks.append("fontFamily")
+
+        return masks, base
+
+
+@dataclass
+class BorderStyle:
+    """Dataclass to represent a Border object for a border along a cell.
+
+    Google documentation: https://developers.google.com/sheets/api/reference/rest/v4/spreadsheets/cells#Border 
+    """
+
+    BORDER_STYLES = ["NONE", "DOTTED", "DASHED", "SOLID", "SOLID_MEDIUM", "SOLID_THICK", "DOUBLE"]
+    COLOR_STYLES = ["rgbColor", "themeColor"]
+    THEME_COLOR_TYPES = ["TEXT", "BACKGROUND", "ACCENT1", "ACCENT2", "ACCENT3", "ACCENT4", "ACCENT5", "ACCENT6", "LINK"]
+
+    position: str
+    style: str
+    color: Color
+    color_style: Any = None
```

Comment:
This should be a `Union` --just stuck the `Any` here while I was finding the right typing operator.